modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/dataSets.spec.ts

```diff
@@ -93,29 +142,356 @@ test.describe('Data Set Fragment', () => {
 					.allInnerTexts()
 			).toEqual(['ID', 'Name', '']);
 		});
+
+		await test.step('Remove Data Set Fragment from the page', async () => {
+			await fdsFragmentPage.editPage({layout});
+
+			await fdsFragmentPage.fdsTableWrapper.click();
+
+			const dataSetFragmentOptionsButton = page
+				.locator('.page-editor__topper__item.tbar-item')
+				.getByLabel('Options');
+			await dataSetFragmentOptionsButton.click();
+
+			const dataSetFragmentOptionsDropdownId =
+				await dataSetFragmentOptionsButton.evaluate((node) =>
+					node.getAttribute('aria-controls')
+				);
+			await page
+				.locator(`#${dataSetFragmentOptionsDropdownId}`)
+				.waitFor();
+
+			await page
+				.locator(`#${dataSetFragmentOptionsDropdownId}`)
+				.getByRole('menuitem', {name: 'Delete'})
+				.click();
+		});
+
+		await test.step('Assert that the Data Set Fragment is not available on the page', async () => {
+			await expect(
+				page.getByText('Place fragments or widgets here.')
+			).toBeInViewport();
+
+			await expect(
+				await fdsFragmentPage.fdsTableWrapper
+			).not.toBeInViewport();
```

Comment:
This one seems to me as a general-purpose fragment removal test, as such, looks more suitable to be in the some `fragment-*` module. We can keep it for completeness, but if this becomes flay for some reason (say, locators change often) we can reconsider removal